src/main/kotlin/io/snyk/plugin/services/CliAdapter.kt

```diff
@@ -48,7 +43,7 @@ abstract class CliAdapter<R>(val project: Project) {
      * Build list of commands for run Snyk CLI command.
      * @return List<String>
      */
-    fun buildCliCommandsList(cmds: List<String>): List<String> {
+    private fun buildCliCommandsList(cmds: List<String>): List<String> {
```

Comment:
What's the reason to make this private and add the other method? It feels very wrong to me, and I would like to understand.

Reply:
Having that call anywhere outside of CliAdapter.execute() leads to have CliCommandPath multiple times in commands passed to CLI -> CLI fails.